homeassistant/components/unifi/switch.py

```diff
@@ -109,6 +136,9 @@ def update_items(controller, async_add_entities, switches):
         LOGGER.debug(
             "New UniFi POE switch %s (%s)", client.hostname, client.mac)
 
+    if switches_off:
+        switches_off = []
```

Comment:
What does this do?

Reply:
Makes sure mechanics that should only be run on setup runs more than once

Reply:
We're not using this variable in this function after this line. Setting `switches_off` to a list, won't affect the reference that we passed in to this function.

Reply:
Ok like this?